src/bundles/files.js

```diff
@@ -172,17 +171,16 @@ export default {
      Action Creators
      ============================================================ */
 
-  doAddFiles: (files) => async ({ dispatch, store, getIpfs }) => {
+  doAddFiles: (files) => async ({ dispatch, getIpfs }) => {
     const ipfs = getIpfs()
     const { streams } = await filesToStreams(files)
 
     for (const stream of streams) {
-      const fileId = shortid.generate()
       const fileName = stream.name
       const fileSize = stream.size
 
       const file = {
-        [fileId]: {
+        [fileName]: {
```

Comment:
I know you're right when saying that having the filename as the id is a non-issue, but I still think it's best to have a shortid as the identifier.

Reply:
Yeah, but if you have two different files with the same name in different folders and upload them both, only one will stick around.

Let's get this PR merged with this reverted and I'll find a solution to that 😉